bugbug/model.py

```diff
@@ -24,7 +27,9 @@ def __init__(self, lemmatization=False):
         else:
             self.text_vectorizer = TfidfVectorizer
 
-        self.undersampling_enabled = True
+        self.undersampling_enabled = False
```

Comment:
Instead of having variables such as this, let's see if we can make each model decide what undersampling/oversampling to use directly in the model implementation (e.g. by adding the sampler to the pipeline for each model, or by having a `self.sampler` property for each model).